### ISSUE TITLE
GO:0043200    response to amino acid

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#20825]
could the term 
GO:0043200    response to amino acid
have the parent "response to nutrient"

and could the exact synopnym "response to amino acid stimulus" be the primary term name (then it is really clear  to the casual reader that this is replete nitrogen).

I need these term sto extends TOR and MAPK signalling pathway terms which have different outcomes depending whether nitrogen is plentiful or during nitrogen starvation.

**Comment by ValWood** [comment]
![Uploading Screenshot 2021-02-08 at 17.45.56.png…]()

**Comment by ukemi** [comment]
Is this term meant to cover amino acids that are neurotransmitters? PMID:32867295

**Comment by ValWood** [comment]
Well I wondered about that, but as the exact synonym is
"response to amino acid stimulus" that is really only used in a nutrient stimulus context isn't it?

We definitely need  away to  extend annotations in this context because sensing amino acid levels determines cell  growth/size.

**Comment by pgaudet** [comment]
I dont think this is always in the context of nutrients - for eg l-DOPA is a child

**Comment by ValWood** [comment]
Isn't L-DOPA an amino-acid derivative?

I notice CHEBI have a classification for proteinogenic and none proteinogenic amino acids. This might be a useful classification for GO. Only proteinogenic amino acids would be in the nutrient branch? At least as far as I am aware these are the only ones involved in starvation signalling.

**Comment by pgaudet** [comment]
To be more accurate biologically, GO:0043200 response to amino acid should be obsoleted, because it groups amino acids with unrelated functions. 

- response to nutrient levels make sense
- synaptic transmission also makes sense
It doesn't make sense to group those at a high level. What do you think ?

**Comment by ValWood** [comment]
I agree.
